Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["plonky2x/core", "plonky2x/derive", "rustx"]
+members = ["plonky2x/core", "plonky2x/derive", "rustx", "api"]
```

Comment:
This is not really an API, it's a client to communicate to an api

Reply:
`succinctx-client`?

Reply:
Updated to `succinctx-client`